docs/_articles/en/user-guide/remote-development.md

```diff
@@ -1,21 +1,26 @@
 ---
-title: Remote Development - Containers
+title: Remote Development
 lang: en
 ---
 
 ## Overview
 
-Salesforce Extension for VS Code supports remote development and allows you to use a container as a full-featured development environment. You can open a project mounted into the container and edit with full code completions, code navigation, debugging, and more.
+Salesforce Extension for VS Code supports remote development and allows you to use a container as a full-featured development environment. You can open a project mounted into a Docker container or using WSL2 and edit with full code completions, code navigation, debugging, and more.
```

Comment:
would spell out like this per my web search: Windows Subsystem for Linux (WSL) 2